com.dxx_redux.D1X-Redux.json

```diff
@@ -0,0 +1,88 @@
+{
+  "id": "com.dxx_redux.D1X-Redux",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "d1x-redux",
+  "rename-desktop-file": "d1x-redux.desktop",
+  "rename-icon": "d1x-redux",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--share=ipc",
+    "--device=all"
+  ],
+  "modules": [
+    "shared-modules/SDL/sdl12-compat.json",
+    "shared-modules/linux-audio/fluidsynth2.json",
+    "SDL_mixer-git.json",
+    "shared-modules/glu/glu-9.json",
+    "shared-modules/glew/glew.json",
+    "shared-modules/physfs/physfs.json",
+    {
+      "name": "D1X-Redux",
+      "buildsystem": "cmake-ninja",
+      "subdir": "d1",
+      "config-opts": [
+          "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+          "-DXDGCONFIG=1"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/dxx-redux/dxx-redux/archive/refs/tags/v1.0.tar.gz",
+          "sha256": "500d70a1c6e3391317b12b7714c2539fb31438e99d32d6016013026749e26bea",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 1,
```

Comment:
I doubt this project-id is acurate.

Reply:
I forgot about that, I'll change it to the proper id.